skrf/network.py

```diff
@@ -1709,6 +1709,75 @@ def stability(self) -> npy.ndarray:
         2 * npy.abs(self.s[:, 0, 1]) * npy.abs(self.s[:, 1, 0]))
         return K
 
+    @property
+    def max_stable_gain(self) -> npy.ndarray:
+        """
+        Maximum stable power gain in linear value
+
+        .. math::
+
+                msg = |S_{21}| / |S_{12}|
+
+        Returns
+        -------
+        msg : :class:`numpy.ndarray` of shape `f`
+
+        References
+        ----------
+        M. S. Gupta, "Power gain in feedback amplifiers, a classic revisited," in IEEE Transactions on Microwave Theory and Techniques, vol. 40, no. 5, pp. 864-879, May 1992, doi: 10.1109/22.137392.
+        """
+        assert self.nports == 2, "Maximum stable gain is only defined for two ports"
+        msg = npy.abs(self.s[:, 1, 0]) / npy.abs(self.s[:, 0, 1])
+        return msg
+    
+    @property
+    def max_gain(self) -> npy.ndarray:
+        """
+        Returns maximum stable power gain for K < 1 or Maximum avalable power gain for K >= 1 in linear value
```

Comment:
isn't it the "Maximum Available Gain" instead?

Reply:
Thank you for the suggestion.
If K > 1, max_gain returns the maximum available gain. On the other hand, if K <= 1, max_gain returns the maximum stable gain. This behavior is the same as the max_gain() function of Keysight's ADS. Thus, I defined the property name as simple as max_gain.